site/content/en/docs/Installation/upgrading.md

```diff
@@ -128,7 +128,74 @@ upgrades.
    between the Agones controller being recreated and GameServers being deleted doesn't occur, and GameServers can end up stuck in erroneous states.
 1. Start and complete your control plane upgrade(s).
 1. Start and complete your node upgrades.
-1. Scale your Fleets back up and/or recreate your GameServers. 
+1. Scale your Fleets back up and/or recreate your GameServers.
 1. Run any other tests to ensure the Agones installation is still working as expected.
 1. Close your maintenance window.
 7. Congratulations - you have now upgraded to a new version of Kubernetes! 👍
+
+{{% feature publishVersion="1.41.0" %}}
+## SDK Proto Compatibility Guarantees as of Agones v1.41.0
```

Comment:
```suggestion
## SDK Proto Compatibility Guarantees
```

As per above, I think we need a _why_ header and description - and probably have a alert box of some kind saying something like "we're start work on in-place upgrades as of version 1.41.0, it's currently Alpha... etc etc"

WDYT?

Reply:
This is in place upgrades adjacent. The guarantees are still valid even if the user does not use in place upgrades, although it is necessary for in place upgrades to have a guaranteed window of compatibility.